partner_multi_relation/models/res_partner_relation.py

```diff
@@ -153,5 +160,13 @@ def _check_relation_uniqueness(self):
                 ]
             if record.search(domain):
                 raise ValidationError(
-                    _("There is already a similar relation with " "overlapping dates")
+                    _(
```

Comment:
@NL66278 This is the way:

```suggestion
                    _("There is already a similar relation with " "overlapping dates"
                    "%(left_partner)s already has a %(connection_type)s"
                        " relation with %(right_partner)s with overlapping dates",
                        left_partner=record_name(record.left_partner_id),
                        connection_type=record.type_id.display_name,
                        right_partner=record_name(record.right_partner_id),
                    )
```

Reply:
Now I understand. Did not know this new functionality. I applied the change you suggested. Thank you!